### ISSUE TITLE
Not everyone uses express...

### ISSUE THREAD

**Issue by joseym** [ArkeologeN/node-linkedin#26]
I'd like to use this library but it seems like it's built explicitly for express.js or a similar framework.

I'm currently unable to return an access token, because the framework i'm using doesn't have a `redirect` method, so the `authorize()` does nothing for me.

**Comment by ArkeologeN** [comment]
@joseym Agreed. Do you think the below method could work for you?

```javascript
if (res.redirect) {
   res.redirect(url);
} else {
   res.writeHead(302, {'Location': url});
   res.end();
}
```

**Comment by joseym** [comment]
maybe make res optional all together, and if it doesn’t exist then return
the url.







*Josey Phone: 918.926.6678 | Gtalk: me@joseymorton.com <me@joseymorton.com>
Twitter: @joseym <http://www.twitter.com/joseym> LinkedIn: Josey Morton
<http://www.linkedin.com/profile/view?id=59813483> Github: joseym
<http://www.github.com/joseym> Blog: Tumblr <http://tumblr.joseymorton.com>*

**Comment by joseym** [comment]
Resolved via bf2d9cf0bdcac5410f55d8ed9172d5e652158bfc
